packages/farms/constants/1.ts

```diff
@@ -279,6 +279,13 @@ export const farmsV3 = defineFarmV3Configs([
     token1: ethereumTokens.stg,
     feeAmount: FeeAmount.MEDIUM,
   },
+  {
+    pid: 41,
```

Comment:
let's move it in front of pid 40. The order matters in the farm config

Reply:
sure